gitlab/util.go

```diff
@@ -124,6 +137,21 @@ func StringIsGitlabVariableName() schema.SchemaValidateFunc {
 	}
 }
 
+func StringIsGitlabVariableType() schema.SchemaValidateFunc {
+	return func(v interface{}, k string) (s []string, es []error) {
+		value, ok := v.(string)
```

Comment:
Expose the good function directly instead of returning it

Reply:
Hey thanks for the suggestion the result is shorter indeed.
I did it like this because I imitated how it was done (that is obviously no excuse ;-) ) I took the liberty to correct those occasions also so that this anti-pattern will not creep in again.